### PR TITLE
Find vulnerabilities sanity test

### PR DESCRIPTION
**Pull request by amirbenun**
### Summary of your changes
Add sanity test for CNVM:
Iterate over `cloud_security_posture.vulnerabilities` index to make sure it is populated.

### Related Issues
- Resolves: https://github.com/elastic/cloudbeat/issues/700

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: tests/commonlib/io_utils.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
 
 def get_events_from_index(
     elastic_client,
-    index_name: str,
```
Comment (oren-zohar) [inline tests/commonlib/io_utils.py:22]: update docstring for removed parameter

File: tests/commonlib/utils.py [hunk 2]
```diff
@@ -293,7 +290,7 @@ def get_findings(elastic_client, config_timeout, query, sort, match_type):
             )
             result = current_result
             break
-        time.sleep(1)
+        time.sleep(5)
```
Comment (oren-zohar) [inline tests/commonlib/utils.py:293]: can you put it into a const with a meaningful name please?

File: tests/integration/tests/test_sanity_checks.py [hunk 3]
```diff
@@ -88,7 +89,28 @@ def test_cspm_findings(elastic_client, match_type):
...
-    query, sort = elastic_client.build_es_must_match_query(must_query_list=query_list, time_range="now-24h")
+    query, sort = cspm_client.build_es_must_match_query(must_query_list=query_list, time_range="now-24h")
 
-    results = get_findings(elastic_client, CONFIG_TIMEOUT, query, sort, match_type)
+    results = get_findings(cspm_client, CONFIG_TIMEOUT, query, sort, match_type)
+    assert len(results) > 0, f"The resource type '{match_type}' is missing"
+
+
+@pytest.mark.sanity
+def test_cnvm_findings(cnvm_client):
```
Comment (oren-zohar) [inline tests/integration/tests/test_sanity_checks.py:99]: Just for consistency and clarity, I would keep it in the same format (which means we need to update `test_data` on line 15) but your call
```suggestion
@pytest.mark.sanity
@pytest.mark.parametrize("match_type", tests_data["cnvm"])
def test_cnvm_findings(cnvm_client):
```

File: tests/configuration.py [hunk 4]
```diff
@@ -7,6 +7,9 @@
 from munch import Munch
 from loguru import logger
 
+FINDINGS_INDEX_PATTETRN = "*cloud_security_posture.findings*"
+VULNERABILITIES_INDEX_PATTETRN = "*cloud_security_posture.vulnerabilities*"
```
Comment (oren-zohar) [inline tests/configuration.py:11]:
```suggestion
FINDINGS_INDEX_PATTERN = "*cloud_security_posture.findings*"
VULNERABILITIES_INDEX_PATTERN = "*cloud_security_posture.vulnerabilities*"
```

**Review by oren-zohar — CHANGES_REQUESTED** [review]

**Comment by amirbenun** [comment]
Working workflow:
https://github.com/amirbenun/cloudbeat/actions/runs/5520102753/jobs/10066260670

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**